examples/nlp/bert_glue_pytorch/startup-hook.sh

```diff
@@ -1 +1,2 @@
 pip install transformers==2.8.0 scikit-learn==0.22.2.post1
+pip install sentencepiece==0.1.91
```

Comment:
Do you need to put this before installing transformers?

Reply:
I re-ran the experiment and it worked with this order of installation.